pkg/storage/installation.go

```diff
@@ -30,6 +31,9 @@ type Installation struct {
 
 // InstallationSpec contains installation fields that represent the desired state of the installation.
 type InstallationSpec struct {
+	// SchemaType indicates the type of resource imported from a file.
```

Comment:
Do we need to have a migration step to populate values for this field for an existing database?

Reply:
SchemaType is not required and supports an empty string for a value, so adding a new optional field doesn't impact existing records.

Reply:
I think it's a bit confusing since the value is manually set in the code but not required in the database. If in the future some other area of the code path references the SchemaType, then it's hard to trace at what point during the code execution the value is populated. That's my main concern here

Reply:
How about I populate the schemaType during apply if it's unset? Or do you only see a full db migration as the acceptable solution?

Reply:
populating it during apply for installation record sounds good to me

Reply:
Thanks! I really appreciate you thinking of this before I messed it up. 😊